File: examples/TanksMiniGameProject/minigame/__temp/__scripts/src/System/TimeZone.js
Bridge.assembly("minigame-adaptor-project", function ($asm, globals) {
    "use strict";

    Bridge.define("System.TimeZone", {
        statics: {
            props: {
                CurrentTimeZone: {
                    get: function () {
                        throw new System.Exception("not impl");
                    }
                }
            }
        },
        ctors: {
            get $ctorDefault() { return this.ctor },
            ctor: function () {
                this.$initialize();
                throw new System.Exception("not impl");
            }
        },
        methods: {
            ToLocalTime: function (time) {
                throw new System.Exception("not impl");
            }
        }
    });
});

File: unity-plugin/script-export/sdk/System~/TimeZone.cs
namespace System {
    public class TimeZone {
        public static TimeZone CurrentTimeZone {
            get {
                throw new Exception("not impl");
            }
        }

        private TimeZone() {
            throw new Exception("not impl");
        }

        public DateTime ToLocalTime(DateTime time) {
            throw new Exception("not impl");
        }
    }
}